src/Expression/ForClasses/DependsOnlyOnTheseNamespaces.php

```diff
@@ -21,14 +21,18 @@ public function __construct(string ...$namespace)
         $this->namespaces = $namespace;
     }
 
-    public function describe(ClassDescription $theClass): Description
+    public function describe(ClassDescription $theClass, string $because): Description
     {
         $desc = implode(', ', $this->namespaces);
 
-        return new PositiveDescription("should depend only on classes in one of these namespaces: $desc");
+        if ('' !== $because) {
```

Comment:
Maybe we can add the because as a second argument of the Positive Description constructor, so that we don't need to repeat the if on the empty because everywhere?

Reply:
+1 for @fain182 suggestion

Reply:
@fain182so are you suggesting to change the PositiveDescription like this?

```
class PositiveDescription implements Description
{
    /** @var string */
    private $description;

    public function __construct(string $description, string $because)
    {
        $this->description = $description;
        if ('' !== $because) {
            $this->description .= ' because' . $because
       }
    }
```

Reply:
Exactly... 
(maybe we could refactor a little more around PositiveDescription and NegativeDescription but we will think about it in another PR)

Reply:
@fain182 I move the because concatenation logic into the PositiveDescription